helevents/utils.py

```diff
@@ -45,6 +49,55 @@ def delete_user_and_gdpr_data(
             ]
         )
 
+    minimum_event_end = timezone.now() - timezone.timedelta(
+        days=settings.GDPR_API_DELETE_EVENT_END_TRESHOLD_DAYS
```

Comment:
Don't worry, 3/4 is still a pretty good result! 😄  

```suggestion
        days=settings.GDPR_API_DELETE_EVENT_END_THRESHOLD_DAYS
```

Reply:
Yes, I consider that an almost victory 🥈